front-end/src/components/settings-view/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './settings.module.scss';
+import PropTypes from 'prop-types';
+
+const Button = ({active, children, onClick}) => (
+    <a 
+        href='' 
+        onClick={e => {
```

Comment:
use the newest pointer events.
https://reactjs.org/blog/2018/05/23/react-v-16-4.html

You are coding for touchscreen. Click is out-of-fashion